mozilla_bitbar_devicepool/configuration.py

```diff
@@ -241,29 +241,27 @@ def configure_projects(update_bitbar=False):
         logger.info('{}: configuring test file'.format(log_header))
         file_name =  project_config.get('test_file')
         if file_name:
-            bitbar_files = get_files(name=file_name, inputtype='test')
+            bitbar_files = get_files(name=file_name)
```

Comment:
Should we remove the inputtype from https://github.com/bclary/mozilla-bitbar-devicepool/blob/master/mozilla_bitbar_devicepool/files.py#L31 ?

Reply:
Done.